views/options.html

```diff
@@ -23,12 +23,12 @@ <h1>ZAP Browser Extension</h1>
         <td>As set in the ZAP API options</td>
       </tr>
       <tr>
-        <td><label>Enable</label></td>
+        <td><label>Enable </label></td>
         <td><input type="checkbox" id="zapenable"/></td>
       </tr>
-      <tr>
-        <td><label>Record</label></td>
-        <td><input type="checkbox" id="zaprecordingactive"/></td>
+      <tr >
+        <td><label>Close window handle when when recording is stopped </label></td>
```

Comment:
The when is duplicated but I think `Automatically close the browser in the recorded scripts.` would be more clear.